base_comment_template/migrations/13.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 NextERP Romania SRL
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    # Not tested
+    properties = env["ir.property"].search(
+        [
+            (
+                "fields_id",
+                "=",
+                env.ref("base.field_res_partner_property_comment_template_id").id,
+            )
+        ]
+    )
+    if properties:
+        for template in properties.mapped("value_reference"):
+            template_id = template.value_reference.split(",")[-1]
```

Comment:
I think this would probably break, as you are ultimately trying to access `value_reference` of the `value_reference` of `ir.property` (and the latest will be a string as you are using `mapped()`). It would probably be something like:
```suggestion
        for template in properties.mapped("value_reference"):
            if not template:
                continue
            template_id = template.split(",")[-1]
```

Example in Odoo console:
```python
>>> env["ir.property"].search([]).mapped("value_reference")
[False, False, False, False, False, False, False, False, False, False, False, False, False, False, False, False, False, False, False, False, False, False, False, False, False, False, False, False, False, 'product.pricelist,1', False, 'account.payment.term,4', 'product.pricelist,1', 'product.pricelist,1', 'product.pricelist,1', 'account.payment.term,4', 'account.payment.term,4', 'account.payment.term,4', 'account.payment.term,7', 'account.payment.term,7', 'product.pricelist,1', 'product.pricelist,1', 'product.pricelist,1', 'account.payment.term,7', 'account.payment.term,7', 'account.payment.term,7', 'account.payment.term,7', 'account.payment.term,7', 'account.payment.term,7', 'account.payment.term,4', 'product.pricelist,1', 'product.pricelist,1', 'product.pricelist,1', 'product.pricelist,1', 'product.pricelist,1', 'product.pricelist,1', 'product.pricelist,1', 'account.payment.term,4', 'account.payment.term,4', 'account.payment.term,4', 'account.payment.term,4', 'account.payment.term,4', 'account.payment.term,4', 'account.payment.term,4', 'account.payment.term,7', 'product.pricelist,1', 'product.pricelist,1', 'product.pricelist,1', 'account.payment.term,7', 'account.payment.term,7', 'account.payment.term,7', 'account.account,193', 'account.account,188', 'account.account,357', 'account.account,481']
>>> env["ir.property"].search([]).mapped("value_reference")[-1]
'account.account,481'  # This is a string
>>> env["ir.property"].search([]).mapped("value_reference")[-1].value_reference
Traceback (most recent call last):
  File "<console>", line 1, in <module>
AttributeError: 'str' object has no attribute 'value_reference'
>>> 
```